testsuite/tests/kuadrant/authorino/operator/clusterwide/test_wildcard_collision.py

```diff
@@ -0,0 +1,46 @@
+"""
+Test for wildcard collisions with clusterwide authorino
+"""
+
+import pytest
+
+from testsuite.openshift.objects.auth_config import AuthConfig
+
+
+# pylint: disable = unused-argument
+@pytest.fixture(scope="module")
+def authorization(authorino, blame, openshift, module_label, envoy):
+    """In case of Authorino, AuthConfig used for authorization"""
+    auth = AuthConfig.create_instance(openshift, blame("ac"), "*.redhat.com", labels={"testRun": module_label})
```

Comment:
Ideally this should not contain redhat.com as the cluster can run on any domain, we should get the wildcard from openshidt itself